src/commands/exec.js

```diff
@@ -49,23 +49,48 @@ function printCodeData(codeData) {
     console.log(output);
 }
 
+function promisify(value) {
+    return Promise.resolve().then(() => value);
```

Comment:
Doesn't have `Promise.resolve(value)` have the same meaning here? The function is already called and its return value is passed here, so if it throws, it's already thrown anyway.